packages/package-json-lint/package.json

```diff
@@ -33,8 +33,7 @@
     "compile": "babel src --out-dir lib --copy-files --ignore src",
     "jest": "jest",
     "lint": "eslint --ext .js,.jsx --ignore-path ../../.eslintignore .",
-    "lint:js:fix": "eslint --ext .js,.jsx . --fix",
-    "postinstall": "npm run compile"
```

Comment:
Just curious why we don't need this script.

Reply:
`postinstall`? It is redundant as the project already gets compiled at the monorepo level and regnerates the `lib` which already exists in the released package.

Reply:
Ok I thought this might be useful for package-level development. But we can just run the compile script separately if needed.

Reply:
The other issue is that this can cause installation failures for consumers if we are not publishing `src` as it'll try to run this script during `npm install`. So this is something we would want to figure out once we normalize package-level development. But that would be a separate discussion/PR.